### ISSUE TITLE
[DropDownButton] raises accessibility errors

### ISSUE THREAD

**Issue by svetq** [telerik/kendo-angular#2791]
**Describe the bug**
The Wave accessibility tool reports the following two errors for an icon dropdown button:
![image](https://user-images.githubusercontent.com/20015190/75980372-02788600-5eeb-11ea-8d8f-205b75b6fdb5.png)

**To Reproduce**
Please check the following example:
https://stackblitz.com/edit/angular-8nyvpf-yytbft?file=app/app.component.ts

**Comment by Raisolution** [comment]
Issue cannot be reproduced. Tested with Wave, ChromeVox and NVDA.
